client/src/components/Home/Filters.js

```diff
@@ -0,0 +1,30 @@
+//ToDo: Add label to the left side of the filters
+//ToDo: Add search icon button to search with filters --> Collect filter and only if the user presses search apply filter
+//ToDo: Make it possible to clear a filter
+
+import React from 'react';
+import 'bootstrap/dist/js/bootstrap.bundle';
```

Comment:
What exactly is this? Why are you importing bootstrap her?

Reply:
Removed, left from a previos implementation version